tests/test_span.py

```diff
@@ -40,6 +40,9 @@ def test_baggage():
     span.set_baggage_item('X_y', '123')
     assert span.get_baggage_item('X_y') == '123'
     assert span.get_baggage_item('x-Y') is None
+    span.set_baggage_item('x-Y', None).set_baggage_item('z', None)
```

Comment:
Why is the first one `x-Y`? It is already None. I am not sure what you're trying to test with that.

I suggest adding something like `set_baggage_item('nonExistingKey', None)` to test that it won't cause an error.